docs/advanced_usage.rst

```diff
@@ -101,7 +101,7 @@ threads::
             time.sleep(1)
 
     with tracer.trace("main_thread") as span:
-        thread = threading.Thread(target=_target, args=(span.context,))
+        thread = threading.Thread(target=_target, args=(tracer.current_trace_context(),))
```

Comment:
is this only needed for the threading example?

should we explicitly add a note/comment to the threading section on why it is needed here but not for multiprocessing?

Reply:
updated all references